open-source-hack/index.html

```diff
@@ -455,6 +502,18 @@ <h4 class="title">Frequently Asked Questions</h4>
 										</div>
                                     </div>
 								</div><!-- card -->
+
+								 <div class="card">
+									<div class="card-header" id="headingEight">
+										<a aria-controls="collapseEight" aria-expanded="false" class="collapsed" data-target="#collapseEight" data-toggle="collapse" href="#">I have some more questions related to Open Source Hack/AnitaB.org?</a>
+									</div>
+									<div aria-labelledby="headingEight" class="collapse" data-parent="#accordionExample" id="collapseEight">
+                                        <div class="card-body">
+											<p class="text">Ask us <a href="">here!</a></p>
+										</div>
```

Comment:
For the link of "Ask us here." Use the questions stream in zulip: https://anitab-org.zulipchat.com/#narrow/stream/223070-questions

Reply:
Done! Also updated the text a little!